partner_credit_limit_history/views/res_partner_view.xml

```diff
@@ -18,6 +18,7 @@
                                 context="{'default_partner_id': active_id, 'default_amount': credit_limit}"
                                 type="action"
                                 string="Change Limit"
+                                class="btn btn-secondary ml-2"
```

Comment:
MIG PR should only focus on migration and fixing blocking issues. But I think we can keep this small UI improvement.